src/ingest_validation_tools/directory_validator.py

```diff
@@ -3,6 +3,38 @@
 from fnmatch import fnmatch
 from importlib import import_module
 
+def default_factory(base_path, assay_type):
+    """
+    default validator factory
+    """
+    return DefaultDirectoryContentValidator(base_path, assay_type)
+
+
+DIRECTORY_CONTENT_VALIDATOR_FACTORY = default_factory
+
+
+class DirectoryContentValidator(object):
+    def __init__(self, base_path, assay_type):
+        self.base_path = base_path
+        self.assay_type = assay_type
+
+    def collect_errors(self):
+        """
+        returns string or None.  Multiple errors are to be reported
+        via a single multi-line string.
+        """
+        return None
+
+
+class DefaultDirectoryContentValidator(DirectoryContentValidator):
+    """
+    Implements backward compatible calls to content validation modules
```

Comment:
This is backward compatibility to the stubs I had created? Unless you see you see yourself using them, they don't need to (and shouldn't) stay around.

Reply:
Yeah, that's backward compatibility.  I don't object to removal, but if the factory plug-in has not been externally set it still has to work sensibly.

Reply:
wrt lint and formatting, I'm not using a proper IDE for these edits so I apologize for breaking PEP8 & etc.  Thanks for fixing things up.  I was running test_dataset_examples.sh but I'm not set up to run travis locally.